libbeat/docs/outputs-list.asciidoc

```diff
@@ -0,0 +1,38 @@
+// TODO: Create script that generates this file. Conditional coding needs to
+// be preserved.
+
+//# tag::outputs-list[]
+
+* <<elasticsearch-output>>
+* <<logstash-output>>
+* <<kafka-output>>
+ifndef::no-output-redis[]
+* <<redis-output>>
+endif::[]
+* <<file-output>>
+* <<console-output>>
+* <<configure-cloud-id>>
+
+//# end::outputs-list[]
+
+//# tag::outputs-include[]
+include::{libbeat-outputs-dir}/elasticsearch/docs/elasticsearch.asciidoc[]
+
+include::{libbeat-outputs-dir}/logstash/docs/logstash.asciidoc[]
+
+include::{libbeat-outputs-dir}/kafka/docs/kafka.asciidoc[]
+
+include::{libbeat-outputs-dir}/redis/docs/redis.asciidoc[]
+
+include::{libbeat-outputs-dir}/fileout/docs/fileout.asciidoc[]
+
+include::{libbeat-outputs-dir}/console/docs/console.asciidoc[]
+
+include::outputs/output-cloud.asciidoc[]
+
+//REVIEWERS: Should the cloud ID topic live somewhere under libbeat/outputs. If
+//so, where? Or should it continue to live somewhere in the docs directory?
```

Comment:
I wonder if we want Cloud support to live under Outputs. Maybe it makes sense to move cloud support one level up in the docs hiearchy?

Reply:
I agree, but I'm concerned that the configuration section is already horribly bloated (it's hard to find stuff in there). I want to move some of the "how to" topics out of that section. "Send events to Elastic Cloud" might be a good candidate. I agree with your feedback here, but I'm going to defer this for now because I think I need to improve the book structure before it makes sense to move this topic.